EternalBAND.DomainObjects/ApiContract/LoginInputContract.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace EternalBAND.DomainObjects.ApiContract
+{
+    public class LoginInputContract
+    {
+        public string Username { get; set; }
+        public string Password { get; set; }
+        public bool RememberMe { get; set; } = false;
```

Comment:
to-do cookie ui

Reply:
There are several options to hold information 

- Local Storage ( when browser tab closed it will be disposed)
- Session storage ( when browser windows closed it will be disposed)
- Cookies ( It will be persistent till user cleans the cookies manually ) 

So we can trust cookies since they are long time lived elements over browser. Since backend is trusting JWT token and looking for **Authorization** header in request, do not need to read or set Cookies. Only Front end will read/modify cookies.

Also we can hold tokens in cookie as well as hashed or BASE 64 encoded.

Reply:
- Local Storage: Not true
- Sessin storage: Half true
- Cookie: Half true

Pls look this [doc](https://stackoverflow.com/questions/19867599/what-is-the-difference-between-localstorage-sessionstorage-session-and-cookies) 
![image](https://github.com/user-attachments/assets/9c2d6ca2-c480-499d-9632-919486a926cc)

Also yes as we discuss before "to-do cookie ui" is written for the FE.  And yes we will encrypt our whole user infos etc or the cookies or storages etc.